core/src/main/java/org/frankframework/stream/Message.java

```diff
@@ -358,56 +357,51 @@ private void preserveToDisk(boolean deepPreserve) throws IOException {
 	@Deprecated
 	@Nullable
 	public Object asObject() {
-		return request;
+		return messageCleanupAction.request;
 	}
 
 	public boolean isBinary() {
-		if (request instanceof SerializableFileReference reference) {
+		if (messageCleanupAction.request instanceof SerializableFileReference reference) {
```

Comment:
Ik vraag me af of je dit niveau van koppeling wilt hebben - als je wilt weten of je Message binary is, zou je toch niet de cleanup action nodig moeten hebben?

Reply:
Nee, wil je niet. Ik ben er zelf ook niet blij mee, en het PR is ook alweer terug gezet in "draft" en we gaan het misschien gewoon sluiten.

Misschien eventueel nog een keer op een andere manier doen -- eigenlijk wil je dat de MessageCleanupAction ook echt iets opschoont, maar daarvoor moet je toegang hebben tot de gegevens van de Message die de action nu niet heeft. Ik zag nog geen mooie oplossing dus voorlopig laten we het maar even.